### PR TITLE
Update session affinity docs

### PR DESCRIPTION
**Pull request by ColeBennett**

### CODE REVIEW

File: content/load-balancing/understand-basics/session-affinity.md [hunk 1]
```diff
@@ -8,25 +8,25 @@ weight: 17
...
 
-### "header"
+### header
 
-The initial origin selection is stable and based on specific HTTP headers found in the first request. Subsequent requests, by clients with the same HTTP headers to the same load balancer, will be sent to the same origin server for the duration of the session and as long as the origin server remains healthy. If the origin server becomes unhealthy, a different stable origin server will be used until the original origin server recovers. A new origin server is calculated and used only after the session has not been used for `session_affinity_ttl` seconds.
+On the first request to a proxied load balancer, a session key based on the configured HTTP headers is generated, encoding the request headers used for storing in the load balancer session state which origin the request will be forwarded to. Subsequent requests to the load balancer with the same headers will be sent to the same origin server, for the duration of the session and as long as the origin server remains healthy. If the session has been idle for the duration of session affinity TTL seconds or the origin server is unhealthy, then a new origin server is calculated and used.
```
Comment (ColeBennett) [inline content/load-balancing/understand-basics/session-affinity.md:29]: Updating to match the API docs

File: content/load-balancing/understand-basics/session-affinity.md [hunk 2]
```diff
@@ -86,4 +88,5 @@ You can enable one of three options:
 
 - **None**: No failover will take place and errors may show to your users.
 - **Temporary**: Traffic will be sent to other origin(s) until the originally pinned origin is available.
-- **Sticky**: The session affinity cookie is updated and subsequent requests are sent to the new origin moving forward as needed.
\ No newline at end of file
+- **Sticky**: The session affinity cookie is updated and subsequent requests are sent to the new origin moving forward as needed.
```
Comment (ColeBennett) [inline content/load-balancing/understand-basics/session-affinity.md:97]: Updating to match the API docs

File: content/load-balancing/understand-basics/session-affinity.md [hunk 3]
```diff
@@ -42,12 +48,13 @@ Enable Session Affinity when you [create or edit a load balancer](/load-balancin
 
 If you enable Session Affinity, choose one of the following options:
 
-- **By Cloudflare cookie only**: Sets a `cflib` cookie to track the associated origin web server
-- **By Cloudflare cookie and Client IP fallback**: Sets a `cflib` cookie, but also uses the client IP address when no session affinity cookie is provided
+- **By Cloudflare cookie only**: Sets a `cflib` cookie to track the associated origin web server.
+- **By Cloudflare cookie and Client IP fallback**: Sets a `cflib` cookie, but also uses the client IP address when no session affinity cookie is provided.
+- **By HTTP header**
```
Comment (patriciasantaana) [inline content/load-balancing/understand-basics/session-affinity.md:53]:
```suggestion
- **By HTTP header**.
```

Comment (patriciasantaana) [inline content/load-balancing/understand-basics/session-affinity.md]:
```suggestion
- **By Cloudflare cookie only**: Sets a `cflib` cookie to track the associated origin web server.
- **By Cloudflare cookie and Client IP fallback**: Sets a `cflib` cookie, but also uses the client IP address when no session affinity cookie is provided.
- **By HTTP header**.
```

**Review by patriciasantaana — APPROVED** [review]

**Pull request merged**